res/sections/90-scripts.tex

```diff
@@ -0,0 +1,35 @@
+\chapter{Script in R}
+
+L'esecuzione di diversi comandi ripetitivi in \texttt{R} a volte può rivelarsi 
+estremamente ripetitiva, e per questo esiste la possibilità di creare quelli che 
+hanno il nome di \textit{script}, ovvero corte sequenze di codice salvate in un 
+file che possono essere richiamate a piacimento con un semplice comando.
+La stesura di \textit{script} quindi ci aiuta la stesura di codice e ci aiuta a 
+velocizzare il nostro lavoro. Per far eseguire questi \textit{snippet} sarà 
+necessario usare \texttt{Rscript}, un comando da terminale apposito o 
+\texttt{RStudio}, l'IDE fatto apposta per programmare in \texttt{R}.
+
+\paragraph*{Rscript} Apriamo un terminale\footnote{Che siamo su Windows, 
+GNU/Linux o MacOS non cambia} nella stessa cartella dove si trova lo script, e 
```

Comment:
Proprio brutto `Che siamo su Windows, GNU/Linux o MacOS non cambia` consiglio `Indifferentemente in Windows, GNU/Linux o MacOS`